docs/GettingStarted.md

```diff
@@ -45,7 +45,10 @@ These scripts are run (you guessed it) **every time** grate executes :)  More in
 
 ## Directory run order
 
-grate processes the files in a standard set of directories in a fixed order for deterministic processing.  Folders are run in the following order:
+By default, grate processes the files in a standard set of directories in a fixed order for deterministic processing.  
+As of grate 1.4, the folder structure is [fully customisable](ConfigurationOptions/FolderConfiguration.md).
```

Comment:
Again, maybe a gentle nudge here to stick with the defaults unless you know you need otherwise?